lib/files/node.ts

```diff
@@ -296,4 +328,34 @@ export abstract class Node {
 		}
 	}
 
+	/**
+	 * Update the attributes of the node
+	 *
+	 * @param attributes The new attributes to update
+	 */
+	update(attributes: Attribute) {
+		// Proxy the attributes to update the mtime on change
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		this._attributes = new Proxy(this.cleanAttributes(attributes) || {} as any, this.handler)
```

Comment:
Empty object is always true, so this should not be needed (talking about the ` ||  { }` )